### ISSUE TITLE
emoji.json missing emojis

### ISSUE THREAD

**Issue by nitely** [Ranks/emojione#260]
Following emojis exists as assets but are not included in the `emoji.json` file. They seem to be letters (with no modifier), numbers (with no modifier) and unicode-9 emojis.

```
0023
002a
0030
0031
0032
0033
0034
0035
0036
0037
0038
0039
1f1e6
1f1e7
1f1e8
1f1e9
1f1ea
1f1eb
1f1ec
1f1ed
1f1ee
1f1ef
1f1f0
1f1f1
1f1f2
1f1f3
1f1f4
1f1f5
1f1f6
1f1f7
1f1f8
1f1f9
1f1fa
1f1fb
1f1fc
1f1fd
1f1fe
1f1ff
1f57a
1f57a-1f3fb
1f57a-1f3fc
1f57a-1f3fd
1f57a-1f3fe
1f57a-1f3ff
1f5a4
1f5e8
1f6d1
1f6d2
1f6f4
1f6f5
1f6f6
1f919
1f919-1f3fb
1f919-1f3fc
1f919-1f3fd
1f919-1f3fe
1f919-1f3ff
1f91a
1f91a-1f3fb
1f91a-1f3fc
1f91a-1f3fd
1f91a-1f3fe
1f91a-1f3ff
1f91b
1f91b-1f3fb
1f91b-1f3fc
1f91b-1f3fd
1f91b-1f3fe
1f91b-1f3ff
1f91c
1f91c-1f3fb
1f91c-1f3fc
1f91c-1f3fd
1f91c-1f3fe
1f91c-1f3ff
1f91d
1f91d-1f3fb
1f91d-1f3fc
1f91d-1f3fd
1f91d-1f3fe
1f91d-1f3ff
1f91e
1f91e-1f3fb
1f91e-1f3fc
1f91e-1f3fd
1f91e-1f3fe
1f91e-1f3ff
1f920
1f921
1f922
1f923
1f924
1f925
1f926
1f926-1f3fb
1f926-1f3fc
1f926-1f3fd
1f926-1f3fe
1f926-1f3ff
1f927
1f930
1f930-1f3fb
1f930-1f3fc
1f930-1f3fd
1f930-1f3fe
1f930-1f3ff
1f933
1f933-1f3fb
1f933-1f3fc
1f933-1f3fd
1f933-1f3fe
1f933-1f3ff
1f934
1f934-1f3fb
1f934-1f3fc
1f934-1f3fd
1f934-1f3fe
1f934-1f3ff
1f935
1f935-1f3fb
1f935-1f3fc
1f935-1f3fd
1f935-1f3fe
1f935-1f3ff
1f936
1f936-1f3fb
1f936-1f3fc
1f936-1f3fd
1f936-1f3fe
1f936-1f3ff
1f937
1f937-1f3fb
1f937-1f3fc
1f937-1f3fd
1f937-1f3fe
1f937-1f3ff
1f938
1f938-1f3fb
1f938-1f3fc
1f938-1f3fd
1f938-1f3fe
1f938-1f3ff
1f939
1f939-1f3fb
1f939-1f3fc
1f939-1f3fd
1f939-1f3fe
1f939-1f3ff
1f93a
1f93b
1f93b-1f3fb
1f93b-1f3fc
1f93b-1f3fd
1f93b-1f3fe
1f93b-1f3ff
1f93c
1f93c-1f3fb
1f93c-1f3fc
1f93c-1f3fd
1f93c-1f3fe
1f93c-1f3ff
1f93d
1f93d-1f3fb
1f93d-1f3fc
1f93d-1f3fd
1f93d-1f3fe
1f93d-1f3ff
1f93e
1f93e-1f3fb
1f93e-1f3fc
1f93e-1f3fd
1f93e-1f3fe
1f93e-1f3ff
1f93f
1f940
1f942
1f943
1f944
1f945
1f946
1f947
1f948
1f949
1f950
1f951
1f952
1f953
1f954
1f955
1f956
1f957
1f958
1f959
1f95a
1f95b
1f95c
1f95d
1f95e
1f960
1f961
1f985
1f986
1f987
1f988
1f989
1f98a
1f98b
1f98c
1f98d
1f98e
1f98f
1f990
1f991
23cf
```

**Comment by caseyahenson** [comment]
@nitely Our UC9 artwork was pre-released in v2.2.0 but not officially supported in our data (emoji.json). Our official UC9 release will be in a couple of week and will address this issue.

**Comment by nitely** [comment]
@caseyahenson Great :+1: 

`1f93b` *shortname* is missing. Should I open an issue?
